streaming_form_data/targets.py

```diff
@@ -164,3 +165,33 @@ def on_data_received(self, chunk: bytes):
     @property
     def value(self):
         return self._hash.hexdigest()
+
+
+class S3Target(BaseTarget):
+    """
+    S3Target implemented using smart_open library. Chunked upload to the S3 target.
+    """
+
+    def __init__(self, file_path, mode, transport_params=None, **kwargs):
+        super().__init__(**kwargs)
+        self._file_path = file_path
+        self._mode = mode
+        self._transport_params = transport_params
+        self.data_received_size = 0
+        self._f_out = None
```

Comment:
Let's rename this to `self._fd` to keep things consistent with the other targets (eg. `FileTarget`).